### PR TITLE
Update CI workflow for C++ SDK

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
Update CI workflow for C++ SDK to enable lifecycle management of GitHub workflows in C++ SDK

### CODE REVIEW

File: src/cpp-sdk/workflows/ci.yml [hunk 1]
```diff
@@ -28,34 +28,56 @@ on:
     branches:
       - main
 jobs:
+  conan-package:
+    runs-on: ubuntu-22.04
+    container: ghcr.io/eclipse-velocitas/devcontainer-base-images/cpp:v0.3
+    name: Check Conan package
+    strategy:
+      matrix:
+        host: ["linux_x86_64"]
```
Comment (MP91) [inline src/cpp-sdk/workflows/ci.yml:37]: should we also test arm now?

Comment (BjoernAtBosch) [reply src/cpp-sdk/workflows/ci.yml:37]: Are there arm runners available now?

Comment (MP91) [reply src/cpp-sdk/workflows/ci.yml:37]: no, but x-compilation should work or?

Comment (BjoernAtBosch) [reply src/cpp-sdk/workflows/ci.yml:37]: Was not able to get that working easily. Please approve.

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**